### PR TITLE
add slow-links-in-page-content.md

### PR DESCRIPTION
**Pull request by garethnic**

### CODE REVIEW

File: content/links/slow-links-in-page-content.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Having slow links in your page content ruins the user experience. You will risk user engagement, declining search engine ranking and possibly reduced income.
+
+# How do I fix this ?
+
+* Reduce HTTP requests.
```
Comment (Johanndutoit) [inline content/links/slow-links-in-page-content.md]: Don't think these quite apply here. As this is how long it takes the server to serve me the page. So I'm only worrying about time to get to and load the initial page from the server. No CSS / JS.

Comment (garethnic) [inline content/links/slow-links-in-page-content.md]: Would these be fine:

* Use as less media (images, video) as possible.
* Make sure caching is enabled, even at page level.
* Make sure links aren't redirecting as this adds to the loading time.

Comment (Johanndutoit) [inline content/links/slow-links-in-page-content.md]: The last 2 yes, but "Use as less media (images, video) as possible." would not fit here. As this only checks links and not assets. There is a separate rule for slow assets.

**Comment by Johanndutoit** [comment]
Sweet and simple, I likes :)

**Pull request merged**